units/crons.go

```diff
@@ -1112,6 +1112,23 @@ func PopulateSpawnhostExpirationCheckJob() amboy.QueueOperation {
 	}
 }
 
+func PopulateVolumeExpirationCheckJob() amboy.QueueOperation {
+	return func(ctx context.Context, queue amboy.Queue) error {
+		volumes, err := host.FindVolumesWithNoExpirationToExtend()
+		if err != nil {
+			return err
+		}
+
+		catcher := grip.NewBasicCatcher()
+		for _, v := range volumes {
```

Comment:
I'd do for idx := range volumes { queue.Put(ctx, NewVolumeExpirationJob(ts, &voumes[idx])

Reply:
np.
Why is it better that way?

Reply:
done